src/routes/businessFinance.js

```diff
@@ -18,15 +18,22 @@ const {
   addDiscountToBusinessFinance,
 } = require("../controllers/businessFinance");
 const { isAuthorized } = require("../middlewares/auth");
+const { BUSINESS_DEFINITION } = require("../constants/pages");
+const { CREATE, UPDATE } = require("../constants/rest");
+const getResourceBusinessIdInCreate = require("../middlewares/auth/utils/getResourceBusinessId/getResourceBusinessIdInCreate");
+const getResourceBusinessIdForFinance = require("../middlewares/auth/utils/getResourceBusinessId/getResourceBusinessIdForFinance");
 
 //parameters
 // router.param("businessFinanceId", getBusinessFinanceIdById);
 
 //all of actual routes
 //create route
+
 router.post(
   "/",
-  isAuthorized(null, null),
+  isAuthorized(BUSINESS_DEFINITION, CREATE, {
```

Comment:
Business can be created/updated/deleted only by super admin. so add a new page `BUSINESS_FINANCE` in `constants/pages`.

Reply:
also update it wherever applicable